README-Starexec.md

```diff
@@ -0,0 +1,44 @@
+## Building and running on Starexec VM
+
+1. The VM:
+```
+VM: https://www.starexec.org/vmimage/
+user: root
+passwd: St@rexec
+```
+2. Install newer CMake:
+```shell
+git clone 'https://gitlab.kitware.com/cmake/cmake.git'
+cd cmake
+./configure
+gmake
+make install
+```
+3. Install Mata
+```shell
+git clone 'https://github.com/VeriFIT/mata.git'
+cd mata
+vim CMakeLists.txt
+# ... comment out the following two lines:
+#       include(CodeCoverage)
+#       setup_target_for_coverage(${PROJECT_NAME}_coverage tests coverage)
```

Comment:
This should not be necessary. If it is, we did something wrong in Mata CMake.

Reply:
True. The problem is that the Starexec readme is obsolete and I don't have time to run it inside starexec vm again.

Reply:
I will make an issue for this, and we will fix it later.

Reply:
I do not think there is any point in updating readme for starexec, it was only for smt-comp, and they are supposed to move away from starexec next year.

Reply:
Ok, lets keep it as it is